src/pudl/metadata/resources/eia860.py

```diff
@@ -641,14 +674,10 @@
         "etl_group": "eia860",
     },
     "_core_eia860__cooling_equipment": {
-        "description": (
-            "Information about cooling equipment at generation facilities, "
-            "from EIA-860 Schedule 6D.\n\n"
-            "Note: This table has been cleaned, but not harvested with other "
-            "EIA 923 or 860 data. The same variables present in this table "
-            "may show up in other _core tables in other years. Once this table "
-            "has been harvested, it will be removed from the PUDL database."
```

Comment:
This should either be in `additional_details` or `usage warnings` or flag it as something we might want to remove in the metadata spreadsheet.